product-service/src/getAllProducts.ts

```diff
@@ -0,0 +1,18 @@
+import { getAllProducts as getAllProductsService } from "./services/products";
+import { winstonLogger } from "./utils/winstonLogger";
+import { errorResponse, successResponse, responseInterface } from "./utils/apiResponseBuilder";
+
+export const getAllProducts: ( event, _context ) => Promise<responseInterface> = async (event, _context) => {
```

Comment:
types should be from capitalised. responseInterface -> `ResponseInterface`

Reply:
do we need _context here?

Reply:
Done in e900383